lib/reek/tree_walker.rb

```diff
@@ -18,24 +18,35 @@ module Reek
   #
   # :reek:TooManyMethods: { max_methods: 29 }
   class TreeWalker
-    def initialize(smell_repository, exp)
-      @smell_repository = smell_repository
-      @exp = exp
+    private_attr_accessor :element
```

Comment:
I will continue doing that until I slayed every last attribute declaration that is NOT at the top of the file ;)